Library/Formula/gnuradio.rb

```diff
@@ -0,0 +1,193 @@
+class Gnuradio < Formula
+  desc "SDK providing the signal processing runtime and processing blocks"
+  homepage "http://gnuradio.squarespace.com/"
+  url "http://gnuradio.org/releases/gnuradio/gnuradio-3.7.8rc1.tar.gz"
+  sha256 "a2228c8cba0194924c7549e37085b5c3837c3282ac96f5bf2392c07cbaed0895"
+
+  # Replace -lpython with -undefined dynamic_lookup in linker flags.
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!